sections/kubecontext.zsh

```diff
@@ -52,11 +56,13 @@ spaceship_kubecontext() {
     fi
   done
 
+  kube_context=" via ($kube_context)"
```

Comment:
To be honest this feels a bit wrong, maybe it's better to have two separate sections, one for version, and one for context? With the current code you have a non-configurable `"via"` prefix, and always-visible version AND context without the ability to hide one of these.

Reply:
I deleted version displaying.